Vikta-UI-testing-sample/src/main/java/com/griddynamics/qa/vikta/uitesting/sample/pageObjects/AddressesPage.java

```diff
@@ -0,0 +1,19 @@
+package com.griddynamics.qa.vikta.uitesting.sample.pageObjects;
+
+import java.util.List;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+public class AddressesPage {
+
+  @FindBy(css = "#tblAddresses > tbody > tr:last-child > td:nth-child(7)")
+  private WebElement lastAddedAddressNickname;
+
+  public WebElement getLastAddedAddressNickname() {
```

Comment:
it's better to have a method that will accept some argument that will let us choose which address we want to select